index.js

```diff
@@ -490,14 +490,37 @@ exports.init = function (sbot, config) {
 
     function next(accept) {
       getConfirm(invite_id, accept, function (err, confirm) {
-        if(!confirm)
-          connectFirst(invite, function (err, rpc) {
-            if(err) return cb(err)
-            rpc.peerInvites.confirm(accept, function (err, confirm) {
-              //TODO: store confirms for us in the state.
-              cb(err, confirm)
-            })
+        if (err) return cb(err)
+        if(confirm) {
+          console.error('ssb-peer-invites: Invite already confirmed')
+          return // mix: send something else back in cb for client side?
+        }
+
+        connectFirst(invite, function (err, rpc) {
+          if(err) return cb(err)
+          rpc.peerInvites.confirm(accept, function (err, confirm) {
+            //TODO: store confirms for us in the state.
+            if (err) return cb(err)
+
+            pull(
+              invite.pubs,
+              paramap(
+                function (addr, cb) {
+                  sbot.gossip.connect(addr, function (err, something) {
+                    if (err) cb(false)
+                    else cb(true)
```

Comment:
did you mean `cb(null, true)` here? I think this will always show the error condition in the `pull.collect` cb

Reply:
yeah that makes more sense. I did go look at pull-paramap to check the docs, and when I found those missing detail I looked at the source code and it was too dense so I made an educated guess